spec/lib/edition_filter_spec.rb

```diff
@@ -203,6 +203,24 @@
         expect(editions).to eq([edition])
       end
     end
+
+    context "when the edition's document type is tagged as pre release" do
+      it "includes the edition for users with pre_release_features permission" do
+        edition1 = create(:edition)
+        edition2 = create(:edition, :pre_release)
+        pre_release_user = build(:user)
+        editions = described_class.new(pre_release_user).editions
+        expect(editions).to match_array([edition1, edition2])
+      end
+
+      it "excludes the edition for users without pre_release_features permission" do
+        edition1 = create(:edition)
+        _edition2 = create(:edition, :pre_release)
```

Comment:
We shouldn't need to define a variable if it's not used.

Reply:
So it is being used but indirectly. The point here is that there are editions with pre release features that are being excluded. If I were to remove this variable then this test would pass without any of the code changes.

Reply:
Have removed the variable declaration